recipes/r-pdftools/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set version = '1.8' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pdftools
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: pdftools_{{ version }}.tar.gz
+  url:
+    - {{ cran_mirror }}/src/contrib/pdftools_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/pdftools/pdftools_{{ version }}.tar.gz
+  sha256: 5f790c3678eacdc15147a671571c86b5f8d6eb692fa680324bc5d27542e6d0fb
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('cxx') }}      # [not win]
+    - {{native}}toolchain        # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}sed               # [win]
+    - {{posix}}grep              # [win]
+    - {{posix}}autoconf
+    - {{posix}}automake-wrapper  # [win]
+    - {{posix}}automake          # [win]
+    - {{posix}}pkg-config
+    - {{posix}}make
+    - {{posix}}sed  # [win]
+    - {{posix}}coreutils  # [win]
+
+  host:
+    - r-base
+    - r-rcpp >=0.12.12
+    - poppler
+
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
+    - r-rcpp >=0.12.12
+
+test:
+  commands:
+    - $R -e "library('pdftools')"           # [not win]
+    - "\"%R%\" -e \"library('pdftools')\""  # [win]
+
+about:
+  home: https://ropensci.org/blog/2016/03/01/pdftools-and-jeroen (blog) https://github.com/ropensci/pdftools#readme
+    (devel) https://poppler.freedesktop.org (upstream)
```

Comment:
The linter's complaints weren't your fault – it was https://github.com/conda-forge/conda-smithy/pull/871, since fixed.

Now to get the CI re-triggered. There're several url's in your `about/home`, and I suspect that won't work nicely with the homepage on anaconda.org (e.g., https://anaconda.org/conda-forge/nlohmann_json). (If you know that this _does_ work, please let me know and ignore this comment.) Perhaps redistribute the url's among [other available keys](https://github.com/conda/conda-build/blob/86fee985afe63d4f661332f60f6e30c652d2f894/conda_build/metadata.py#L500) or make some other commit to start the CI.

I notice now that the circle-CI's error at your previous commit is that it's not finding libpoppler-cpp at runtime. As conda-forge moves toward conda-build 3, A `run_exports` will probably be added to the `poppler` pkg, so that a `reqs/run` dependency is imposed from a `reqs/host` dependency. But in the meantime, I think you need to add `- {{ pin_compatible('poppler') }}` to `reqs/run` in order to pick up on [the pinning](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L204-L205).